### ISSUE TITLE
write script to take list of blocked email address' from auth0 and delete user accounts from db

### ISSUE THREAD

**Issue by ascott** [participedia/api#1032]

**Comment by jesicarson** [comment]
Hey @ascott have you run the script yet? hopefully you can run it again...
i have just spent the last hour going through the auth0 users list and blocked probably 300 more users, and counting... im only on 2 months back so far in my sweep, and it appears to have started over 9 months ago. some of the spam addresses are effoby, nedoz, protonmail, etc there are about 10 that I can tell right away are spam. however, there are also probably hundreds more that should be deleted but I can't verify that they're spam for sure because I cant check their profile pages - they look like suspicious accounts, but its super hard to tell for sure since many are gmail and outlook accounts but with odd names that sort of look like they could be legit. 
I think this script is a good first step, but Is there any way to beef up spam blocking before they are allowed to create profile pages in the first place? i would say add a captcha or something but we're not sure if we're dealing with people or bots. perhaps we do need that dashboard / admin page we discussed to provide a way for an admin to find a user profile by searching for an email address on the site, if there's no way to stop these people/bots from creating accounts. 
anyway, long story short, ill keep going through this sweep and let you know when ive gone through the entire list and deleted the obvious ones. we can chat more on our call tomorrow.

**Comment by ascott** [comment]
@jesicarson the script is done and on prod, but i have not run it yet.i was planning on running it tomorrow morning, and it can be run again in the future, as more users are blocked via auth0.

about protonmail users, I don't know that all prontonmail email address' are spam. it is a legit encrypted mail provider: https://protonmail.com/, and some spammy users may be using it but i don't think that means all protonmail users are spam.

discussing a better process/handling of spam on our call is a good idea. 👍
